wpimath/src/main/java/edu/wpi/first/math/system/plant/DCMotor.java

```diff
@@ -99,7 +99,7 @@ public double getCurrent(double torqueNm) {
    * @return The torque output.
    */
   public double getTorque(double currentAmpere) {
-    return currentAmpere * KtNMPerAmp;
+    return (currentAmpere - freeCurrentAmps) * KtNMPerAmp;
```

Comment:
This returns a negative torque at 0 A, but at 0 velocity, 0 A actually corresponds to 0 torque. (This and `getSpeed()` are what makes accounting for free current so difficult.)